docs/operator-docs/trigger-workflow.md

```diff
@@ -0,0 +1,78 @@
+---
+id: trigger-workflow
+title: Trigger a Workflow Execution
+hide_title: false
+---
+
+To trigger a workflow, you need to [create a rule](../configuration/data-management-types#create-a-rule). To trigger an ingest workflow, one that requires discovering and ingesting data, you will also need to configure the collection and provider and associate those to a rule.
+
+## Trigger a HelloWorld Workflow
+
+To trigger a [HelloWorld workflow](../data-cookbooks/hello-world) that does not need to discover or archive data, you just need to [create a rule](../configuration/data-management-types#create-a-rule).
+
+You can leave the provider and collection blank and do not need any additional metadata. If you create a `onetime` rule, the workflow execution will start momentarily and you can view its status on the Executions page.
+
+## Trigger an Ingest Workflow
+
+To ingest data, you will need a provider and collection configured to tell your workflow where to discover data and where to archive the data respectively.
+
+Follow the instructions to [create a provider](../configuration/data-management-types#create-a-provider) and [create a collection](../configuration/data-management-types#create-a-collection) and [configure their fields](../data-cookbooks/setup#working-with-data-cookbooks) for your data ingest.
+
+In the rule's additional metadata you can specify a `provider_path` from which to get the data from the provider.
+
+### Example: Ingest data from S3
+
+#### Setup
+
+Assume there are 2 files to be ingested in an S3 bucket called `discovery-bucket`, located in the `test-data` folder:
+- GRANULE.A2017025.jpg
+- GRANULE.A2017025.hdf
+
+Public and protected buckets should exist and be mapped to bucket names in the Cumulus deployment.
```

Comment:
Is this note relevant here or for the collection configuration section below? are you trying to call out that the `discovery-bucket` has to be named in the buckets config?

Reply:
Updated to make this more clear